troubleshooting/Guide to building CCExtractor windows.exe in Visual Studio.md

```diff
@@ -0,0 +1,19 @@
+### **Guide to building CCExtractor windows.exe in Visual Studio 2015+**
+
+Go to: https://github.com/CCExtractor/ccextractor
+
+You will see this page. Press the green “Clone or download” button, and click Open in Desktop. It will take you to Github Desktop, then press clone.
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/build-img1)
+
+Open up your Visual Studio version. Go to the top right and click File -> Open -> Open Project/Solution. Browse to Github -> CCExtractor -> Windows, and select ccextractor.sln, then hit open.
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/build-img2)
+
+The project will open, you can see the tree on the right. Right click the ccextractor project and click build.
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/build-img3)
+
+It should take a few moments, then say build successful:
+
+![alt text](https://github.com/CCExtractor/ccextractor/tree/master/troubleshooting/images/build-img4)
```

Comment:
we already have "https://github.com/CCExtractor/ccextractor/wiki/Installation#windows" I dont think we need this info again